pyasic/miners/backends/luxminer.py

```diff
@@ -146,6 +147,32 @@ async def reboot(self) -> bool:
     async def get_config(self) -> MinerConfig:
         return self.config
 
+    async def upgrade_firmware(self) -> dict:
+        """
+        Upgrade the firmware on a LuxOS miner by calling the 'updaterun' API command.
+
+        Returns: Result of the upgrade process.
+
+        """
+        command = "updaterun"
+
+        try:
+            response = await self.rpc.send_command(command=command)
```

Comment:
Dont have to do this, should call `self.rpc.upgraderun()`, and just have it defined in rpc as 
```
async def upgraderun(self):
    return await self.send_command("upgraderun")
```

Reply:
Done